packages/turf-isobands/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "turf-isobands",
+  "version": "2.0.0",
+  "description": "turf isobands module",
+  "main": "index.js",
```

Comment:
Include `"types":"index.d.ts"` (next to main instead of at the bottom) & `"files": ["index.js", "index.d.ts"]`